packaging/nnstreamer.spec

```diff
@@ -174,22 +175,25 @@ Group:		Multimedia/Framework
 Requires:	capi-nnstreamer-single-new = %{version}-%{release}
 %description -n capi-nnstreamer-single-new-devel
 Developmental kit for Tizen Native new single-shot NNStreamer API.
-%define api -Denable-capi=true
-%else
-%define api -Denable-capi=false
 %endif
 
-# Element restriction in Tizen
+# Define build options
 %if %{with tizen}
+%define enable_tizen -Denable-tizen=true
+%define enable_api -Denable-capi=true
+
+# Element restriction in Tizen
 %define restricted_element	'capsfilter input-selector output-selector queue tee valve appsink appsrc audioconvert audiorate audioresample audiomixer videoconvert videocrop videorate videoscale videoflip videomixer compositor fakesrc fakesink filesrc filesink audiotestsrc videotestsrc jpegparse jpegenc jpegdec pngenc pngdec tcpclientsink tcpclientsrc tcpserversink tcpserversrc udpsink udpsrc xvimagesink ximagesink evasimagesink evaspixmapsink glimagesink theoraenc lame vorbisenc wavenc volume oggmux avimux matroskamux v4l2src avsysvideosrc camerasrc fimcconvert'
 
 %define restriction -Denable-element-restriction=true -Drestricted-elements=%{restricted_element}
 %else
+%define enable_tizen -Denable-tizen=false
+%define enable_api -Denable-capi=false
```

Comment:
Why disable capi without tizen? It can be built without tizen as well.

Reply:
It was disabled in the past, now capi packages only in tizen. Plz see line 148.